apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx

```diff
@@ -137,28 +137,40 @@ export const PatientProfile = () => {
                 <div className="main-body">
                     {patient && summary && <PatientProfileSummary summary={summary} patient={patient} />}
 
-                    <div className="grid-row flex-align-center">
-                        <h6
+                    <div role="tablist" className="grid-row flex-align-center">
+                        <p
+                            role="tab"
```

Comment:
suggestion: The `p` element isn't tabable, these should include `tabindex=0` to make them tabable